frontend/views/partials/books.html

```diff
@@ -2,9 +2,14 @@
 
     	<div class="col-lg-12">
                 <h1 class="page-header">Browse {{page}}</h1>
+      </div>
```

Comment:
@KarimElSheikh 
please add section identifier to know what begins where
like
<!-edit form starts here-->

Html code
<!-edit form ends here-->